tests/foreman/api/test_permission.py

```diff
@@ -30,45 +30,44 @@
 
 from robottelo import ssh
 from robottelo.constants import PERMISSIONS
+from robottelo.datafactory import parametrized
 from robottelo.helpers import get_nailgun_config
 from robottelo.helpers import get_server_software
-from robottelo.test import APITestCase
 
 
-class PermissionTestCase(APITestCase):
+class TestPermission:
     """Tests for the ``permissions`` path."""
 
-    @classmethod
-    def setUpClass(cls):
-        super().setUpClass()
-        cls.permissions = PERMISSIONS.copy()
+    @pytest.fixture(autouse=True)
```

Comment:
You should include `scope='class'` in the fixture decorator arguments. Currently this will copy permissions for every function individually, as `'function'` is the default fixture scope.  Defining it at the class limits its use to test functions within this class - but it will still execute/teardown at a function scope.